src/pages/EditAccount/EditAccountWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import { COLORS } from "@constants/colors";
+import styled from "styled-components";
+
+const EditAccountWrapper = styled.div`
+  width: 100%;
+  padding: 25px 10px 20px 10px;
+  .personalIfo {
+    margin: 10px 0;
+  }
+  .line {
+    background: ${COLORS.blue};
+    margin: 10px 0;
+  }
+  .paragraph {
+    font-size: 18px;
+    font-weight: 400;
+  }
+  .addressIfo {
+    margin: 10px 0;
+  }
+`;
+
+export default EditAccountWrapper;
```

Comment:
why is this file a .tsx format instead of .ts since it is not returning jsx syntax.kindly convert the file back to .ts format